### ISSUE TITLE
Missing local host flipper UI endpoint

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#70274]
### Description

I seem to remember that the Vets-API flipper UI endpoints included the localhost entry. It's missing now.

Under "Vets-API Flipper UI endpoints", please add 

* Local development (http://localhost:3000/flipper/features) (not authenticated)

It might be nice to also include this same list of endpoints on the feature toggles guide page. It's easy to miss the Flipper UI Access link on that page.

### Relevant URLs

- https://depo-platform-documentation.scrollhelp.site/developer-docs/flipper-ui-access#FlipperUIAccess-Vets-APIFlipperUIendpoints
- https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide

### Which type of team are you on? (Platform team, VFS team, or Leadership)

VFS team - Benefits decision reviews

**Comment by johnny-jesensky-adhoc** [comment]
Hi @Mottie, thank you for flagging this. Our team will take a look and work to get these pages updated.

We'll provide updates in this ticket.

**Comment by taylojill** [comment]
Hello @Mottie I updated both pages as requested and attached two pdf docs for your review. Here's a summary:

1. On the Flipper UI Access page: I added "Local development" to Vets-API Flipper UI endpoints bulleted list.
2. On the Feature toggles guide page: I added the updated "Vets-API Flipper UI endpoints" list near top of page, under the Feature toggles list. Let me know if the placement of the list works or if it should be moved. 

If you approve of the updates, I'll get both pages ready for publishing. Thanks @Mottie 

cc @johnny-jesensky-adhoc 


[PWD-Copy of Flipper UI Access-111223-182243.pdf](https://github.com/department-of-veterans-affairs/va.gov-team/files/13639600/PWD-Copy.of.Flipper.UI.Access-111223-182243.pdf)

[PWD-Copy of Feature toggles guide-111223-182346.pdf](https://github.com/department-of-veterans-affairs/va.gov-team/files/13639604/PWD-Copy.of.Feature.toggles.guide-111223-182346.pdf)
